Sources/TuistGenerator/Graph/Graph.swift

```diff
@@ -305,13 +311,44 @@ class Graph: Graphing {
 
         /// Other targets' frameworks.
         let otherTargetFrameworks = findAll(targetNode: targetNode, test: isFramework)
-            .map { DependencyReference.product(target: $0.target.name) }
+            .map { DependencyReference.product(target: $0.target.name, productName: $0.target.productNameWithExtension) }
 
         references.append(contentsOf: otherTargetFrameworks)
 
         return Set(references).sorted()
     }
 
+    func copyProductDependencies(path: AbsolutePath, target: Target) -> [DependencyReference] {
```

Comment:
I think `copyProductDependencies` is a bit ambiguous. What does it mean? What about including a note like [this one](https://github.com/tuist/tuist/pull/515/files#diff-bd14a50b734b61af028ddef07944c580L279) explaining why this method is useful?

Reply:
👍 added a comment to https://github.com/tuist/tuist/pull/515/files#diff-12f3cd68bf1720345626c27c16f99056R108